draft-ietf-taps-interface.md

```diff
@@ -1323,8 +1323,8 @@ or categories of interfaces it wants to `Require`, `Prohibit`, `Prefer`, or
 selection to that single interface, and often leads to less flexible and resilient
 connection establishment.
 
-In contrast to other Selection Properties, this property is a tuple of an
-(Enumerated) interface identifier and a preference, and can either be
+In contrast to other Selection Properties, this property is a set of
+(Enumerated) interface identifier and preference tuples. It can either be
```

Comment:
I find the tuple in the end very hard to read
```suggestion
In contrast to other Selection Properties, this property is a set of
tuples of (Enumerated) interface identifier and preference. It can either be
```